src/components/PopUpWindow.js

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from 'react';
+import Modal from '@material-ui/core/Modal';
+import Backdrop from '@material-ui/core/Backdrop';
+import Fade from '@material-ui/core/Fade';
+import withStyles from '@material-ui/core/styles/withStyles';
+import ScheduleRoundedIcon from '@material-ui/icons/ScheduleRounded';
+import EditRoundedIcon from '@material-ui/icons/EditRounded';
+import MoreHorizRoundedIcon from '@material-ui/icons/MoreHorizRounded';
+import PlayCircleFilledWhiteRoundedIcon from '@material-ui/icons/PlayCircleFilledWhiteRounded';
+
+const styles = theme => ({
+  modal: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  paper: {
+    backgroundColor: theme.palette.background.paper,
+    border: '2px solid #000',
+    boxShadow: theme.shadows[5],
+    padding: theme.spacing(2, 4, 3),
+  },
+});
+
+
+
+class TransitionsModal extends Component {
+    constructor() {
+        super();
+        this.state = {
+            open: false
+        }
+    }
+
+    handleOpen = () => {
+        this.setState({
+            open: true
+        });
+    }
+
+    handleClose = () => {
+        this.setState({
+            open: false
+        });
+    }
+
+    getIcon = () => {
+        switch (this.props.data) {
+            case 'Schedule': 
+                return <ScheduleRoundedIcon onClick={this.handleOpen} />
+
+            case 'Edit':
+                return <EditRoundedIcon onClick={this.handleOpen} />
+
+            case 'More':
+                return <MoreHorizRoundedIcon onClick={this.handleOpen} />
+
+            case 'Play': 
+                return <PlayCircleFilledWhiteRoundedIcon onClick={this.handleOpen} />
+            default: 
```

Comment:
Formerly, you leave a line before new 'case' but before 'default' you didn't leave a line. You should choose one way.